arch/arm/core/aarch64/reset.S

```diff
@@ -115,8 +115,31 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
 	eret
```

Comment:
Slightly off-topic for this review: I added code here under `CONFIG_INIT_STACKS` and `CONFIG_STACK_SENTINEL` to initialize the interrupt stacks / interrupt stack sentinels if appropriate. Otherwise e.g. `kernel stacks` always showed the interrupt stacks as 100% used.

I don't know where the appropriate spot is to put this initialization code; I figured that "when we set the stack pointer to the interrupt stack in the first place" was as good a spot to put it as any (you don't want to do it in C code, because you're then using the stack...)

Reply:
Can you push this change as a PR?

Reply:
Sure, shall do.